services/galley/src/Galley/API/Util.hs

```diff
@@ -881,3 +881,28 @@ instance
     if err' == demote @e
       then throwS @e
       else rethrowErrors @effs @r err'
+
+isTyping ::
```

Comment:
A comment separator (just like before the remote error section) would be nice to have.